docs/cennznet/cennzx.md

```diff
@@ -19,103 +19,125 @@ The following sections contain the module details.
  
 # Storage
  
-### coreAssetId(): `AssetId`
+### coreAssetId(): `u32`
 - **interface**: `api.query.cennzx.coreAssetId`
 - **summary**:   Asset Id of the core liquidity asset 
  
-### defaultFeeRate(): `FeeRate`
+### defaultFeeRate(): `u128`
 - **interface**: `api.query.cennzx.defaultFeeRate`
 - **summary**:   Default trading fee rate 
  
-### liquidityBalance(`ExchangeKey, AccountId`): `Balance`
+### liquidityBalance(`(u32,u32), AccountId32`): `u128`
 - **interface**: `api.query.cennzx.liquidityBalance`
 - **summary**:   Liquidity holdings of a user in an exchange pool. Key: `(core_asset_id, trade_asset_id), account_id` 
  
-### totalLiquidity(`ExchangeKey`): `Balance`
+### totalLiquidity(`(u32,u32)`): `u128`
 - **interface**: `api.query.cennzx.totalLiquidity`
 - **summary**:   Total liquidity holdings of all investors in an exchange. ie/ total_liquidity(exchange) == sum(liquidity_balance(exchange, user)) at all times 
  
 # Extrinsic
  
-### addLiquidity(asset_id: `Compact<AssetId>`, min_liquidity: `Compact<Balance>`, max_asset_amount: `Compact<Balance>`, core_amount: `Compact<Balance>`)
+### addLiquidity(asset_id: `Compact<u32>`, min_liquidity: `Compact<u128>`, max_asset_amount: `Compact<u128>`, core_amount: `Compact<u128>`)
 - **interface**: `api.tx.cennzx.addLiquidity`
-- **summary**:   Deposit core asset and trade asset at current ratio to mint liquidity Returns amount of liquidity minted. 
+- **summary**:   eposit core asset and trade asset at current ratio to mint liquidity eturns amount of liquidity minted. 
```

Comment:
need to check why it is eating up the first letter.. D from posit..R from returns for cennzx doc